File: input_list_devices.lua
-- List input devices

hg = require("harfang")

hg.InputInit()

names = hg.GetMouseNames()
print('Mouse device names: ' .. table.concat(names, ','))

names = hg.GetKeyboardNames()
print('Keyboard device names: ' .. table.concat(names, ','))

names = hg.GetGamepadNames()
print('Gamepad device names: ' .. table.concat(names, ','))

File: input_list_devices.py
# List input devices

import harfang as hg

hg.InputInit()

names = hg.GetMouseNames()
print('Mouse device names: ' + ','.join(names))

names = hg.GetKeyboardNames()
print('Keyboard device names: ' + ','.join(names))

names = hg.GetGamepadNames()
print('Gamepad device names: ' + ','.join(names))
